config/config.go

```diff
@@ -144,6 +149,14 @@ func (c Config) GetResourceRateLimit() ResourceRateLimit {
 	}
 }
 
+func (c Config) GetPodGCDeleteDelay() time.Duration {
+	if c.PodGCDeleteDelay == nil {
+		return 30 * time.Second
```

Comment:
Defaulting to 30s, happy to lower but this was somewhat in the middle of the suggested 10s and the 1m we would run in our environment.

Reply:
I think this should be lower - 5s

Reply:
Lowered to 5s by default.